doc/REST-interface.md

```diff
@@ -89,11 +89,13 @@ Only supports JSON as output format.
 * softforks : (array) status of softforks in progress
 
 #### Query UTXO set
-`GET /rest/getutxos/<checkmempool>/<txid>-<n>/<txid>-<n>/.../<txid>-<n>.<bin|hex|json>`
+`GET /rest/getutxos/<TXID>-<N>/<TXID>-<N>/.../<TXID>-<N>.<bin|hex|json>` <br>
```

Comment:
Extra space before each of the two `<br>` tags (maybe cleaner to add another newline instead of the tags)